public/scripts/main.js

```diff
@@ -145,9 +136,6 @@ const gameHandler = (function gameIIFE() {
     game.hidden = false
     const STARTING_REQUIRED_WORD_COUNT = 10
     requiredWords.addWords(STARTING_REQUIRED_WORD_COUNT)
```

Comment:
This adding of words is only needed for the AI, right? Can you fix this up as part of these changes?